.vscode/settings.json

```diff
@@ -4,5 +4,6 @@
     "statusBar.background": "#004699",
     "titleBar.inactiveBackground": "#EC0000",
     "titleBar.inactiveForeground": "#CCC"
-  }
+  },
+  "editor.wordWrapColumn": 80
```

Comment:
Should we have a .vscode settings file in the repo? I feel like we shouldn't

Reply:
How did this come back? It's in my global gitiginore. I'll check it out.